semantics.json

```diff
@@ -338,6 +338,14 @@
             "type": "boolean",
             "optional": true,
             "default": false
+          },
+          {
+            "name": "randomizeBranchingQuestions",
+            "label": "Randomize Branching Questions",
```

Comment:
It is alternatives that are displayed in random order. Questions are always the same. I think we need to ypdate this text.